sdks/python/setup.py

```diff
@@ -95,7 +95,7 @@ def get_version():
 different technologies and user communities.
 '''
 
-RECOMMENDED_MIN_PIP_VERSION = '7.0.0'
```

Comment:
Is this change needed for 2.43.0 ? I wonder if this can result in other regressions.

Reply:
Yes, pip can discover manylinux_2014 wheels from version 19.3.

Reply:
it's harmless, used below in a warning message